srcpkgs/mumble/patches/pipewire-fix.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/src/mumble/PipeWire.cpp b/src/mumble/PipeWire.cpp
```

Comment:
where does this fix come from? can you add a reference to an upstream git commit or PR link or something along those lines?

Reply:
Fix comes fom upstream. Sure I'll add a description.

Reply:
I added the original description of the upstream PR and a link to it.